src/lib/utils/fetcher.tsx

```diff
@@ -17,6 +18,7 @@ export async function _fetch<ResponseType = any>(
         "Content-Type": "application/json",
         Authorization: config.authKey,
       },
+      signal: abortController ? abortController?.signal : null,
       body: JSON.stringify({
```

Comment:
aborting will throw an abort error right, are we treating it as an error or setting that error null?

Reply:
in the case of fetch methods we return data and error (message string), in cases of hook we do not return an error.